scss/components/lfd/small-card.scss

```diff
@@ -344,6 +344,10 @@
     $layoutSelector: '#{$instanceSelector}[data-id="#{$widgetInstanceId}"] #{$layout}';
   }
 
+  #{$instanceSelector} {
+    width: 100%;
```

Comment:
This is not using any value from the user. Why can't this be hardcoded into the LFD CSS files.

Reply:
@tonytlwu  Yes, you are right. I'll close this PR and create a new one for the dynamic-lists repository.